include/crocoddyl/multibody/residuals/contact-wrench-cone.hxx

```diff
@@ -14,24 +14,18 @@ template <typename Scalar>
 ResidualModelContactWrenchConeTpl<Scalar>::ResidualModelContactWrenchConeTpl(boost::shared_ptr<StateMultibody> state,
                                                                              const pinocchio::FrameIndex id,
                                                                              const WrenchCone& fref,
-                                                                             const std::size_t nu)
```

Comment:
We should add this safety check back - is there a reason why they have been removed everywhere?